os/arch/arm/src/amebasmart/amebasmart_irq.c

```diff
@@ -72,6 +71,10 @@ uintptr_t g_irqstack_top[CONFIG_SMP_NCPUS] =
 #endif
 };
 
+#if defined(CONFIG_ARMV7A_DECODEFIQ)
```

Comment:
Is this config CONFIG_ARMV7A_DECODEFIQ disabled by default?

Reply:
Yes, Its disabled in RTL8730e. It will be enabled only if we use fiq.